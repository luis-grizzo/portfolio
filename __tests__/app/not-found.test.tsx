import { render } from '@testing-library/react'

import NotFound from '@/app/not-found'

describe('NotFound', () => {
  beforeEach(() => {
    render(<NotFound />)
  })

  it('should match snapshot', () => {
    const { container } = render(<NotFound />)

    expect(container).toMatchSnapshot()
  })
})
