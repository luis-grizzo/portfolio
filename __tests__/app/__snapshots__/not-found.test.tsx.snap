// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should match snapshot 1`] = `
<div>
  <main
    class="flex flex-col items-center justify-center gap-4 md:gap-8 lg:gap-16 h-[calc(100vh_-_114px_-_88px_-_1px)] container mx-auto mt-[114px] py-8 md:py-16 lg:py-32 px-4"
  >
    <h1
      class="text-4xl sm:text-5xl"
    >
      404
    </h1>
    <p
      class="text-xl sm:text-2xl text-center text-neutral-400"
    >
      It seems like there's nothing here!
    </p>
    <a
      class="flex items-center gap-3 w-fit px-4 py-3 text-base bg-neutral-900/60 backdrop-blur text-neutral-50 border-1 border-neutral-400/10 rounded-lg transition-colors hover:bg-neutral-900/40"
      href="/"
      target="_self"
    >
      Return Home
      <svg
        class="h-4 w-4 "
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M19 7v4H5.83l3.58-3.59L8 6l-6 6 6 6 1.41-1.41L5.83 13H21V7z"
        />
      </svg>
    </a>
  </main>
</div>
`;
